app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None | str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor["name"]}"
+                                     f" is not vaccinated.")

```

Comment:
There is a syntax error in the string formatting. You should use single quotes around the key `name` inside the f-string or escape the double quotes. For example: `f"{visitor['name']} is not vaccinated."`.